openedx/features/course_duration_limits/access.py

```diff
@@ -132,11 +132,18 @@ def register_course_expired_message(request, course):
         )
     else:
         upgrade_message = _('Your access to this course expires on {expiration_date}. \
-                <a href="{upgrade_link}">Upgrade now</a> for unlimited access.')
```

Comment:
fyi @emma-green after looking at this again in this PR I think we needed the HTML pulled out of the translations like in the example here. I forgot about this previously. https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#mako-template-files